src/main/java/org/gridsuite/cases/importer/job/CaseImportLogger.java

```diff
@@ -6,64 +6,108 @@
+            Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(new JdbcConnection(conn));
+            updateLiquibase(database);
+        } catch (DatabaseException e) {
+            throw new RuntimeException(e);
+        }
     }
 
-    public void connectDb(String hostname, int port, String datacenter, String keyspaceName) {
-        connector.connect(hostname, port, datacenter);
-
-        setKeyspaceName(keyspaceName);
-
-        psInsertImportedFile = connector.getSession().prepare(insertInto(this.keyspaceName, FILES_TABLE)
-                .value(FILENAME_COLUMN, bindMarker())
-                .value(ORIGIN_COLUMN, bindMarker())
-                .value(IMPORT_DATE_COLUMN, bindMarker())
-                .build());
-
+    private void updateLiquibase(Database database) {
+        Properties properties = new Properties();
+        try (Liquibase liquibase = new Liquibase("db/changelog/db.changelog-master.yaml",
+                                                 new ClassLoaderResourceAccessor(),
+                                                 database);) {
+            properties.forEach((key, value) -> liquibase.setChangeLogParameter(Objects.toString(key), value));
+            liquibase.update(new Contexts(), new LabelExpression());
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
     }
 
     public boolean isImportedFile(String filename, String origin) {
-        ResultSet resultSet = connector.getSession().execute(selectFrom(this.keyspaceName, FILES_TABLE)
-                .columns(
-                        FILENAME_COLUMN,
-                        ORIGIN_COLUMN,
-                        IMPORT_DATE_COLUMN)
-                .whereColumn(FILENAME_COLUMN).isEqualTo(literal(filename))
-                .whereColumn(ORIGIN_COLUMN).isEqualTo(literal(origin))
-                .build());
-        Row one = resultSet.one();
-        return one != null;
+        try (PreparedStatement preparedStatement = getCurrentConnection().prepareStatement(SELECT_QUERY,
+                                                                                           ResultSet.TYPE_SCROLL_SENSITIVE,
+                                                                                           ResultSet.CONCUR_UPDATABLE)) {
+            preparedStatement.setString(1, filename);
+            preparedStatement.setString(2, origin);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            boolean isImportedFile = resultSet.first();
+            preparedStatement.close();
+
+            return isImportedFile;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
     }
 
     public void logFileAcquired(String fileName, String origin, Date date) {
-        connector.getSession().execute(psInsertImportedFile.bind(fileName, origin, date.toInstant()));
+        PreparedStatement preparedStatement = null;
+        try {
```

Comment:
again ;)
try (PreparedStatement preparedStatement = conn.prepareStatement(INSERT_QUERY); ) {
and remove preparedStatement.close()